Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM resin/rpi-raspbian:wheezy-2015-01-15
+FROM resin/rpi-raspbian:jessie-2015-04-22
```

Comment:
@shaunmulligan Sure - I thought that the original intention was to keep it to a known working version. I can change this.